test_files/reexport.declaration/reexport_named.ts

```diff
@@ -0,0 +1,3 @@
+// reexport_named.ts exports symbols, but doesn't declare any of them, so its .d.ts
```

Comment:
but why? What does this fix?

Reply:
Without my change this test emits clutz aliases which are invalid. Admittedly, I still don't have a great understanding of how TS APIs model all these cases, hence the confusing comments and code (I have no idea what I am doing). But I can tell you that this test is legitimately broken before my change.